pom.xml.next

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In a normal situation, these files are generated in target folder, which is already in .gitignore.
But i agree, it will be safer =)